bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java

```diff
@@ -62,16 +65,19 @@ public void restart() {
 
 	public void state() {
 		try {
-			String where = port.get().map(i -> i.toString()).orElse("-"); //$NON-NLS-1$
-			System.out.println(server.state() + " on port " + where); //$NON-NLS-1$
+			String usedBindAddress = listen.get().orElse("-"); //$NON-NLS-1$
```

Comment:
compound names again

Reply:
Here I am running out of names, it is the same thing in a different type, when I cannot compound I have to hide fields. See updated PR.